### DISCUSSION TITLE
使用下载推理模型 检测偏移

### DISCUSSION THREAD

**Discussion by yuan916** [PaddlePaddle/PaddleOCR#12818] [Q&A]
- 系统环境/System Environment：win10
- 版本号/Version：Paddle：2.5 
- 运行指令/Command Code：python tools/infer/predict_det.py  --image_dir="./doc/imgs_en/img_12.jpg" --det_model_dir="./inference_model/ch_PP-OCRv3_det_infer/"
- 
- 完整报错/Complete Error Message：
![det_res_img_12](https://github.com/PaddlePaddle/PaddleOCR/assets/147596531/8df581f1-8faf-48dd-a7dd-1e18eca65e94)
其他识别也如图：
![det_res_img_10](https://github.com/PaddlePaddle/PaddleOCR/assets/147596531/326060ca-7641-46c3-b736-3a6b6af297e2)

**Comment by SWHL** [comment]
如问题仍然存在，请拉取最新main分支代码，再次尝试
